### DISCUSSION TITLE
slash command migration

### DISCUSSION THREAD

**Discussion by s0r3-glitch** [nextcord/nextcord#347] [Q&A] [answered]
Has anyone made any examples on how to migrate bots over to slash commands yet? I am hoping to update my bot soon to use slash commands but didnt see anything in the documentation when i skimmed over it. 

edit: found pull requests with everything i needed

**Comment by TAG-Epic** [comment, marked as answer]
Docs will be merged soon.

**Reply by Nopus64** [reply]
thanks for it. Happy to switch @ slashes ;)
